src/appshell/view/notationstatusbarmodel.cpp

```diff
@@ -48,7 +48,8 @@ static constexpr int MIN_DISPLAYED_ZOOM_PERCENTAGE = 25;
 static const QMap<ViewMode, ActionCode> ALL_MODE_MAP {
     { ViewMode::PAGE, "view-mode-page" },
     { ViewMode::LINE, "view-mode-continuous" },
-    { ViewMode::SYSTEM, "view-mode-single" }
+    { ViewMode::SYSTEM, "view-mode-single" },
+    { ViewMode::FLOAT, "view-mode-float" },
```

Comment:
Do I understand correctly that float mode will always be in the list, but when we click on it, it will apply only  in dev mode?